npc/re/warps/cities/izlude.txt

```diff
@@ -101,7 +101,9 @@ OnTouch:
 	close2;
 	if (isbegin_quest(21001) == 1)
 		completequest 21001;
-	warp "izlude" + replacestr( strnpcinfo(2), "intro_to_izlude", "" ),196,209;
+	.@map$ = "izlude" + replacestr( strnpcinfo(2), "intro_to_izlude", "" );
+	warp .@map$,196,209;
+	savepoint .@map$,128,142,1,1;
```

Comment:
Why is the savepoint at `128,142` instead of `196,209;`?

Reply:
It's like that on kRO. Respawn point are near kafra NPCs.